sdk/src/feature_set.rs

```diff
@@ -817,6 +817,10 @@ pub mod migrate_config_program_to_core_bpf {
     solana_sdk::declare_id!("2Fr57nzzkLYXW695UdDxDeR5fhnZWSttZeZYemrnpGFV");
 }
 
+pub mod zk_elgamal_proof_program_enabled {
+    solana_sdk::declare_id!("B6ehYoZirDwWHFoQwujCjMcpxTFfvxvWAnHw5wKGu7hT");
```

Comment:
nit-entirely-up-to-you: all of the other zk features start with zk, which makes them easier to find when running `solana feature status` -- any reason to not do that here too?

Reply:
Oh yeah, no good reason. I just grinded a keypair and replaced it!